### ISSUE TITLE
Angle returns as NaN on Android

### ISSUE THREAD

**Issue by willwull** [willwull/circle-slider#3]
I tried this on some Android phones, namely Galaxy S8 and S7, and none of them work properly.
When trying to move the handle in the demo page, the number changes to `NaN`, and the handle doesn't move either.

This could be caused by a number of methods, like `_moveHandle`, `_getRawAngle`, `_getCenter`, or `_radToDeg`, but I don't own any Android devices to test with.

**Comment by SujitSingh** [comment]
@willwull I have recently created one app's page which runs perfectly on Android platform(tested in few android versions).

```
$(".circular-slider").roundSlider({
    sliderType: "min-range",
    handleShape: "dot",
    max: 40,
    min: 0,
    width: 15,
    radius: 100,
    value: 25,
    startAngle: 270,
    handleSize: "+7",
    showTooltip: false,
    editableTooltip: false,
});
```

I am yet to check those methods mentioned by you. 
And BTW, you can emulate any Android OS using [Android Studio](https://developer.android.com/studio/index.html).

**Comment by willwull** [comment]
@sujit77 I'll try out Android Studio! Didn't know that existed actually 😅

**Comment by akshaydwivedi29** [comment]
@willwull , this indeed is a bug on android. happens due to event type (e) in mousehandler function on line 135.
i'm creating a pull request with the fix.

**Comment by willwull** [comment]
Should now be fixed in #13, thanks to @akshaydwivedi29
